README.md

```diff
@@ -56,14 +55,15 @@ We use **data connector** as a general term to represent data source and data de
 Execute the following commands to start pre-built images with all the dependencies:
 
 ```bash
-$ git clone https://github.com/instill-ai/vdp.git
+$ git clone https://github.com/instill-ai/vdp.git && cd vdp
 
-$ cd vdp
+# Build instill/vdp:dev local development image
+$ make build
```

Comment:
Do we need to add `make build` for Quickstart? I thought `make all` will cover it?

Reply:
We probably don't want to build `instill-ai/vdp:dev` everytime when `make all` so `make build` is a step alone now.